AutomatedTesting/Gem/PythonTests/AWS/Windows/aws_metrics/aws_metrics_automation_test.py

```diff
@@ -227,3 +224,13 @@ def test_unauthorized_user_request_rejected(self,
                 halt_on_unexpected=True)
             assert result, 'Metrics events are sent successfully by unauthorized user'
             logger.info('Unauthorized user is rejected to send metrics.')
+
+    def test_clean_up_s3_bucket(self,

```

Comment:
where does it get called?

Reply:
It's a separate test case that will be run automatically by pytest. The function doesn't need to be called manually.